From 7a2cde0816a6c4740cec1b37bd0841d846399f58 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 7 Apr 2016 19:29:33 +0800
Subject: [PATCH] ARM64: dts: rk3399-chrome: delete unused code in dts

We shouldn't need them in here if you are using the coreboot/firmware.
In general, the cmdline/memory/logic_center will be overwrited
since the coreboot will do that.

Change-Id: I3902ff4eb71891b5c6320bed4355992e699e4835
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-chrome.dts | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
index f31ee1c929fa..2e6c9d570471 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-chrome.dts
@@ -47,14 +47,8 @@
 	model = "Rockchip Chrome";
 	compatible ="google,rk3399evb-rev2",  "google,rk3399evb-rev1", "google,rk3399evb-rev0", "rockchip,chrome", "rockchip,rk3399";
 
-	chosen {
-		bootargs = "rootwait cros_debug cros_secure ro root=/dev/mmcblk0p3 init=/sbin/init console=uart,mmio32,0xff1a0000";
-	};
-
-	memory@00000000 {
-		device_type = "memory";
-		reg = <0x0 0x00000000 0x0 0x20000000>;
-	};
+	/delete-node/ vdd-log;
+	/delete-node/ vdd-center;
 };
 
 &mipi_dsi {
-- 
2.35.3

